xbmc/platform/android/activity/AndroidKey.cpp

```diff
@@ -8,15 +8,12 @@
 
 #include "AndroidKey.h"
 
-#include "AndroidExtra.h"
 #include "ServiceBroker.h"
 #include "XBMCApp.h"
-#include "input/Key.h"
```

Comment:
I couldn't track down where the transitive include for https://github.com/xbmc/xbmc/blob/a03dede299ed07de5a8e6e9462fff62858acbf87/xbmc/input/XBMC_keysym.h is coming from, but I think it should be added in explicitly for the use of XBMCK_* defines

Reply:
You are right, we should not rely on headers being included transitively. We must include a header if it's explicitly needed in that file.